crates/uv-fs/src/lib.rs

```diff
@@ -316,27 +316,37 @@ pub struct LockedFile(fs_err::File);
 impl LockedFile {
     /// Inner implementation for [`LockedFile::acquire_blocking`] and [`LockedFile::acquire`].
     fn lock_file_blocking(file: fs_err::File, resource: &str) -> Result<Self, std::io::Error> {
-        trace!("Checking lock for `{resource}`");
+        trace!(
+            "Checking lock for `{resource}` at `{}`",
+            file.path().user_display()
+        );
         match file.file().try_lock_exclusive() {
             Ok(()) => {
                 debug!("Acquired lock for `{resource}`");
                 Ok(Self(file))
             }
             Err(err) => {
                 // Log error code and enum kind to help debugging more exotic failures
-                debug!("Try lock error, waiting for exclusive lock: {:?}", err);
+                // TODO(zanieb): When `raw_os_error` stabilizes, use that to avoid displaying
+                // the error when it is `WouldBlock`, which is expected and noisy otherwise.
+                trace!("Try lock error, waiting for exclusive lock: {:?}", err);
                 warn_user!(
-                    "Waiting to acquire lock for {} (lockfile: {})",
-                    resource,
+                    "Waiting to acquire lock for `{resource}` at `{}`",
                     file.path().user_display(),
                 );
                 file.file().lock_exclusive().map_err(|err| {
                     // Not an fs_err method, we need to build our own path context
                     std::io::Error::new(
                         std::io::ErrorKind::Other,
-                        format!("Could not lock {}: {}", file.path().user_display(), err),
+                        format!(
+                            "Could not acquire lock for `{resource}` at `{}`: {}",
+                            file.path().user_display(),
+                            err
+                        ),
                     )
                 })?;
+
+                debug!("Acquired lock for `{resource}`");
```

Comment:
Should this be `trace!` like the above?

Reply:
I don't think so — the acquired lock message above is debug level. The trace logs are for the acquire lock error message and the check message.

Reply:
It still feels like a lot for `--verbose` IMO.

Reply:
That's fine with me I think, but we'll want to change both.